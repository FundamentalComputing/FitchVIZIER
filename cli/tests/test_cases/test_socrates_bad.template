∀x (H(x) → M(x))
H(socrat)
M(plato)
