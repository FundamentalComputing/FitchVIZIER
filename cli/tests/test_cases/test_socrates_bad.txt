1 | ∀x (H(x) → M(x))
2 | H(socrates)
  |----
3 | H(socrates) → M(socrates)         ∀ Elim: 1
4 | M(socrates)                       → Elim: 3,2
