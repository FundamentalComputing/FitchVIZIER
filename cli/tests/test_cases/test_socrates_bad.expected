The premises of your proof do not match the premises in the proof template.

The conclusion of your proof does not match the conclusion in the proof template.
